NEWS.md

```diff
@@ -1,3 +1,22 @@
+flux-accounting version 0.10.0 - 2021-09-30
+-------------------------------------------
+
+#### Fixes
+
+* Fix bug in add-user where wrong number of arguments were passed to function (#140)
+
+* Fix bug in edit-user where edit made in one user/bank row was reflected in all of a user's rows in the flux-accounting database (#140)
```

Comment:
"was reflected" or "wasn't reflected?" 

You are probably right but I just wanted to make sure. I looked at #140 but from its high level description I wasn't sure.

Reply:
"was reflected" should be the right description - the fix was to make it so that an edit made in one user/bank row was _only_ made in just that one row instead of in multiple rows.

Reply:
Great! I find this additional description is a bit clearer than the original. Perhaps we use that instead? Thoughts?

```suggestion
* Fix bug in edit-user to ensure an edit made in one user/bank row was only made in just that one row instead of in multiple rows in the flux-accounting database (#140)
```

Reply:
Sure! That's a great suggestion; I'll push that change up. Thanks!